spec/workers/atmosphere/vm_load_monitoring_worker_spec.rb

```diff
@@ -6,6 +6,20 @@
     Fog.mock!
   }
 
+  let!(:vm1) { 
+    create(
+      :virtual_machine,
+      ip: '10.100.0.1',
+      monitoring_id: 1,
+      managed_by_atmosphere: true
+    )
+  }
+  let!(:vm2) { create(:virtual_machine, ip: '10.100.0.2', monitoring_id: 2,
+    managed_by_atmosphere: true) }
+  let!(:vm3) { create(:virtual_machine, ip: '10.100.0.3',
+    managed_by_atmosphere: true) }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 20, 34 should be on its own line.